notebooks/official/model_monitoring/model_monitoring.ipynb

```diff
@@ -336,13 +336,14 @@
       "source": [
```

Comment:
<p>use notebook template version</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/859/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>I think we discussed this in a previous review, the region needs to match where the training data lives or model monitoring gets unhappy.</p>

Reply:
got it